core/src/trezor/ui/layouts/tr/__init__.py

```diff
@@ -1135,7 +1135,12 @@ async def confirm_sign_identity(
 
 
 async def confirm_signverify(
-    coin: str, message: str, address: str, verify: bool
+    message: str,
+    address: str,
+    verify: bool,
+    path: str | None = None,
+    account: str | None = None,
+    chunkify: bool = False,
```

Comment:
Will we implement the same `info` (and `chunkify`) functionality for `TR` as well? @Hannsek

(so we would actually use these function parameters)

Reply:
It is already implemented, no? Except of the "SHOW ALL" screen.
![Uploading image.png…]()

Reply:
on TR, it should be like this:
<img width="624" alt="image" src="https://github.com/trezor/trezor-firmware/assets/57008159/47ceb0ca-7c39-4784-a738-3afabed75185">

But we can change it to:
<img width="768" alt="image" src="https://github.com/trezor/trezor-firmware/assets/57008159/4d6cd833-e922-49c6-9dd6-89c6a9f4ac54">
🤔